test/scholar/linear/svm_test.exs

```diff
@@ -0,0 +1,186 @@
+defmodule Scholar.Linear.SVMTest do
+  use Scholar.Case, async: true
+  alias Scholar.Linear.SVM
+  doctest SVM
+
+  defp iris_data do
```

Comment:
I'm not sure this data should be kept here. Isn't this dataset used elsewhere too?

Reply:
Yes, it is also used in logistic regression. I may move both data and splitting function into `test_helper.exs`, WDYT?

Reply:
Actually, there is a `test/support` directory, so I'll add that there ;)